h/services/annotation_sync.py

```diff
@@ -78,13 +82,16 @@ def sync(self, limit):
         if counter.annotation_ids_to_sync:
             self._batch_indexer.index(counter.annotation_ids_to_sync)
 
+        if counter.annotation_ids_to_delete:
+            self._batch_indexer.delete(counter.annotation_ids_to_delete)
```

Comment:
Make an additional HTTP request to Elasticsearch's bulk API for any annotations that need to be deleted. This is in addition to the bulk API request that's already sent for any annotations that need to be (re-)indexed.